.gitpod.yaml

```diff
@@ -0,0 +1,16 @@
+  # List the start-up tasks. Learn more: https://www.gitpod.io/docs/configure/workspaces/tasks
+  tasks:
+    - name: Install Modular & Mojo
+      before: |
+        eval $(command gp env -e)
+      command: |
+        curl https://get.modular.com | MODULAR_AUTH=$MODULAR_AUTH sh -
```

Comment:
How is `$MODULAR_AUTH` set and what is the default value?

Reply:
This file wasn't meant to be there, removed thanks